.github/workflows/dependency-graph.yml

```diff
@@ -1,14 +1,24 @@
-name: Update Dependency Graph
+name: Update Dependency Graph for SBT
 on:
   push:
     branches:
-      - main # default branch of the project
+      - main
+  workflow_dispatch:
```

Comment:
non-blocking: was this left in intentionally?

Reply:
No, it just matches what dep graph integrator pr generator produces now (i.e. without the comment)